src/pudl_archiver/metadata/sources.py

```diff
@@ -287,4 +287,106 @@
         "license_pudl": LICENSES["cc-by-4.0"],
         "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
     },
+    "doeiraec": {
+        "title": "DOE -- IRA Energy Community Data Layers",
+        "path": "https://edx.netl.doe.gov/dataset/ira-energy-community-data-layers",
+        "description": (
+            "This dataset outlines energy communities across the U.S. based off part of the Inflation "
+            "Reduction Act’s (IRA) definition that is focused on census tracts and directly adjoining "
+            "tracts that have had coal mine closures since 1999 or coal-fired electric generating unit "
+            "retirements since 2009."
+        ),
+        "working_partitions": {"years": [2024]},
+        "keywords": sorted(
+            {
+                "energy community",
+                "coal",
+                "mines",
+                "closure",
+                "ira",
+            }
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "epadcejp": {
+        "title": "EPA -- Disadvantaged Community Energy Justice Program Dataset",
+        "path": "https://www.arcgis.com/home/webmap/viewer.html?url=https://geopub.epa.gov/arcgis/rest/services/ejscreen/environmental_climate_justice_program/MapServer&source=sd",
```

Comment:
These paths are for human consumption, so pointing at a top level documentation page for the dataset rather than the map is probably preferable.  Maybe it's this one?
```suggestion
        "path": "https://www.epa.gov/ejscreen/technical-information-and-data-downloads",
```

For the actual archiving, the ArcGIS MapServer API endpoint might be where we want to point people:

https://geopub.epa.gov/arcgis/rest/services/ejscreen/environmental_climate_justice_program/MapServer

The compound URL you have is saying "use the ArcGIS website to display the data available at this MapServer endpoint"